src/lib/kw_db.sh

```diff
@@ -140,6 +140,54 @@ function replace_into()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function updates or insert rows into table of given database,
```

Comment:
I didn't understand why you added this. Could you explain? I think it might not be very connected to this PR, or am I missing something?